### PR TITLE
Handle Venmo Desktop

### PR DESCRIPTION
**Pull request by mnicpt**
Revert settings for Mobile eligibility.

**Testing**
1. Tested Venmo button on iOS in a SFVC
2. Tested Venmo button on Desktop
3. Tested Venmo button on iOS outside of SFVC and successful app switch.

### CODE REVIEW

**Review by elizabethmv — APPROVED** [review]

File: src/zoid/buttons/util.js [hunk 1]
```diff
@@ -77,33 +77,38 @@ export function createVenmoExperiment() : Experiment | void {
...
-    return {
-        enableVenmo: Boolean(isExperimentEnabled || isEnableFundingVenmo)
-    };
+    if (isDevice()) {
+        return {
+            enableVenmo: Boolean((isExperimentEnabled || isVenmoFundingEnabled) && isNativeSupported)
+        };
+    } else {
+        return {
+            enableVenmo: Boolean(isExperimentEnabled || isVenmoFundingEnabled)
```
Comment (gregjopa) [inline src/zoid/buttons/util.js:109]: I don't think `enable-funding=venmo` is suppose to show Venmo Desktop. Venmo Desktop should only be shown from being in the experiment test group.
```suggestion
            enableVenmo: Boolean(isExperimentEnabled)
```

Comment (mnicpt) [reply src/zoid/buttons/util.js:109]: Thanks, @gregjopa

Comment (TNuzzi) [reply src/zoid/buttons/util.js:109]: @gregjopa @mnicpt would there be another way to force Venmo to show?  The `enable-funding=venmo` is convenient for testing, specially for e2e.

File: test/integration/tests/funding/venmo/index.js [hunk 2]
```diff
@@ -77,13 +77,13 @@ describe(`venmo desktop web button test `, () => {
     });
 
 
-    it(`should display button when it's passed in as instrument on desktop web`, (done) => {
+    it(`should not display button when it's passed in as instrument on desktop web`, (done) => {
```
Comment (gregjopa) [inline test/integration/tests/funding/venmo/index.js]: Ideally we would have a test for each scenario (displaying and not displaying). What do you think about using the localstorage opt-in for the tests?

For example, we can use `localStorage.setItem('enable_venmo_desktop', true)` to opt into Venmo Desktop. And we could set this to false for the negative test case. That way the ramping percentage does not impact the test cases.

**Review by gregjopa — APPROVED** [review]

File: test/integration/tests/funding/venmo/index.js (hunk 2 above)
Comment (gregjopa) [reply test/integration/tests/funding/venmo/index.js]: ah ignore this. Just realized that this file is no longer changed.

**Pull request merged**